tensorflow/lite/examples/label_image/label_image_test.cc

```diff
@@ -37,15 +37,15 @@ TEST(LabelImageTest, GraceHopper) {
 
   std::vector<uint8_t> output(606 * 517 * 3);
   resize<uint8_t>(output.data(), input.data(), 606, 517, 3, 214, 214, 3, &s);
-  ASSERT_EQ(output[0], 0x15);
-  ASSERT_EQ(output[214 * 214 * 3 - 1], 0x11);
+  ASSERT_EQ(output[0], 0x0);
+  ASSERT_EQ(output[214 * 214 * 3 - 1], 0x0);
```

Comment:
The issue is with these lines. The input actually *is* uint8 (and not float). Line 31 initializes the `Settings` object to its default values, which assumes `TfLiteType input_type = kTfLiteFloat32;`. So these values `0x15` & `0x11` should not be changed, we should rather set `s.input_type = kTfLiteUInt8` (after importing `tensorflow/lite/c/common.h`).

Reply:
Thanks. Will fix it.

Reply:
Since it was merged. Open a new issue (#37502) seems cleaner to me.